packages/jellyfish-api-core/__tests__/category/loan/listVaults.test.ts

```diff
@@ -103,26 +104,32 @@ describe('Loan listVaults', () => {
     await testing.generate(1)
 
     // List vaults
-    // should be only vaultId1, vaultId2, vaultId3 coz isUnderLiquidation filter default value equals to false
+    // should be only vaultId1, vaultId2, vaultId3, vaultId4 coz isUnderLiquidation filter default value equals to false
```

Comment:
looks like this is not true with lastest cpp side developments. default state is `VaultState::Unknown`